up-l1/README.adoc

```diff
@@ -288,9 +290,9 @@ We can get the message types by reorganizing the table above to the following on
 |===
 | Message Type | Possible resource_id combinations {src_resource_id, sink_resource_id}
 
-| Publish      | {[8000-FFFE], None}
+| Publish      | {[8000-FFFE], None}, {FFFF, None}
 | Notification | {[8000-FFFE], 0}, {FFFF, 0}, {FFFF, FFFF}
-| Request      | {0, [1-7FFF]}, {FFFF, FFFF}
+| Request      | {0, [1-7FFF]}, {FFFF, [1-7FFF]}, {FFFF, FFFF}
```

Comment:
I think the {ANY, ANY} for request is not valid

Reply:
I thought this was also used in ustreamer. @PLeVasseur would you mind helping us double check?

Reply:
This line is not used in the uStreamer.

We don't care what the resource IDs are for either the sink or source on Request/Response/Notification messages are when we are routing them through.

So that means the really important line for the uStreamer is that last line:
> uStreamer listens for all notifications, requests, and responses

See where we add a "forwarding listener" [here](https://github.com/eclipse-uprotocol/up-streamer-rust/blob/d4ab17fbc745705290e29e5251fa8e9fcbffe943/up-streamer/src/ustreamer.rs#L158-L159) and then the definitions of these functions [here](https://github.com/eclipse-uprotocol/up-streamer-rust/blob/d4ab17fbc745705290e29e5251fa8e9fcbffe943/up-streamer/src/ustreamer.rs#L31-L49).

Reply:
If I understand correctly, then we still need `{ANY, ANY}` in the uStreamer case, right?

Reply:
We need `{ANY, ANY}` for being able to get all notification, request, and response. We don't have a need to get specifically only Requests with `{ANY, ANY}`.

Reply:
oh, let me make it clear. The table here is the reorganized one from the table above. That means if we have `{ANY, ANY}`, the indicated message type will be notification, request and response.
![image](https://github.com/user-attachments/assets/8fbcdf8d-78f4-4996-ab1f-67f7b29a6d03)

Therefore in this case, we should get the request if the combination of resource ID is `{ANY, ANY}`. I guess the table might be a little hard to understand. Feel free to suggest a better form (I don't have a better idea...)

Reply:
Ah okay, I understand what you mean now.

So yes -- source and sink UUri filters for resource IDs of {FFFF, FFFF} could mean any of Notification, Request, or Response.